src/cubehelix.rs

```diff
@@ -1,7 +1,11 @@
 #![allow(clippy::many_single_char_names)]
 
 use crate::Color;
-use std::f32::consts as f32;
+use core::f32::consts as f32;
+
+#[cfg(not(feature = "std"))]
+#[allow(unused_imports)]
```

Comment:
I'm still unclear why `#[allow(unused_imports)]` is necessary here. It seems in some phase of the build the `std` impls of the methods in `F32Ext` are available.

This occurred with the original `libm::F32Ext` as well. I asked about it here but never got to the root of it:

https://github.com/rust-lang/libm/issues/158